proxy-node-shared/src/main/java/uk/gov/ida/notification/contracts/EidasSamlParserResponse.java

```diff
@@ -20,18 +17,13 @@
 
     @NotNull
     @JsonProperty
-    @JsonSerialize(using = X509CertificateSerializer.class)
-    @JsonDeserialize(using = X509CertificateDeserializer.class)
-    private X509Certificate connectorEncryptionPublicCertificate;
+    private String connectorEncryptionPublicCertificate;
 
     @JsonProperty
     @NotBlank
     private String destination;
 
-    private EidasSamlParserResponse() {
```

Comment:
Same as above, I think it's needed for serialisation.

Reply:
Restored 👍